src/core/src/op/asinh.cpp

```diff
@@ -2,77 +2,61 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "ngraph/op/asinh.hpp"
-
-#include <string>
-#include <vector>
+#include "openvino/op/asinh.hpp"
 
+#include "element_visitor.hpp"
 #include "itt.hpp"
-#include "ngraph/op/util/elementwise_args.hpp"
-#include "ngraph/runtime/host_tensor.hpp"
-#include "ngraph/type/element_type.hpp"
 #include "openvino/reference/asinh.hpp"
 
-using namespace std;
-using namespace ngraph;
+namespace ov {
```

Comment:
With using std in the cpp files we get a mix of styles: some places use std and have a code like `std::something` some places doesn't use and code looks like `something`.
Without using std we have one style inside one file.

Reply:
I've changed it because of some automation.
It also is considered that `using namespace std` (or any other) as bad practice to use it in .cpp on global scope. What is recommended to use it in very limited scope (function, code block) to have control over it. For global use, import  only required features from namespace.